spec/api/v2/schemas.rb

```diff
@@ -43,6 +43,7 @@ module Schemas
         supported_profile: SUPPORTED_PROFILE,
         system: SYSTEM,
         tailoring: TAILORING,
+        create_tailoring: CREATE_TAILORING,
```

Comment:
```suggestion
        created_tailoring: CREATED_TAILORING,
```

Reply:
Semantically not correct, it's the schema that is needed when you create a tailoring. Not for a tailoring that was already createD.

Reply:
`create_tailoring` does not seem fitting, as it describes more the action, which is not what is described in scheme.

What about `new_tailoring`?

Reply:
It's not a new, it is connected to the creation.

Reply:
What's wrong with CREATE_TAILORING?

Reply:
I might consider swapping it to TAILORING_CREATE.

Reply:
I think that would be consistent with `policy_update` so I'll do that.